src/run.ts

```diff
@@ -4,11 +4,16 @@ import { program } from 'commander'
 import {Kysely, MigrationResultSet, Migrator, NO_MIGRATIONS} from 'kysely'
 
 function showResults({ error, results }: MigrationResultSet) {
+  if (results) {
+    results.forEach((it) => console.log(`> ${it.status}: ${it.migrationName} (${it.direction})`))
+    if (!results.length) {
```

Comment:
I think this is more clear about the purpose

```suggestion
    if (results.length === 0) {
```

Reply:
Addressed in [9c1e704](https://github.com/acro5piano/kysely-migration-cli/pull/10/commits/9c1e70485f5266761c47698053d208fdbdac26f7)